lib/modules/manager/sbt/extract.ts

```diff
@@ -289,14 +295,34 @@ export function extractPackageFile(
 ): PackageFileContent | null {
   let parsedResult: Ctx | null = null;
 
-  try {
-    parsedResult = scala.query(content, query, {
-      vars: {},
-      deps: [],
-      registryUrls: [REGISTRY_URLS.mavenCentral],
-    });
-  } catch (err) /* istanbul ignore next */ {
-    logger.warn({ err }, 'Sbt parsing error');
+  if (_packageFile === 'project/build.properties') {
```

Comment:
```suggestion
  if (packageFile === 'project/build.properties') {
```
rename parameter too